gh-badges/lib/color.js

```diff
@@ -29,6 +42,8 @@ function normalizeColor(color) {
     return undefined
   } else if (color in namedColors) {
     return color
+  } else if (color in aliases) {
+    return aliases[color]
```

Comment:
Could we use `resolvedAliases` here instead? It's been a long day and my brain is struggling sorry 😄

Reply:
The idea of this function is to take the input color and return something that is normalized, yet still human readable and pleasant to test. For named colors it passes through their names; they don't translate to their hex equivalents. So, the appropriate response for the aliases is the sort of normalized / canonical color name (e.g. `red` -> `red`; `critical` -> `red`).

Reply:
Ah okay, thanks for the explanation. I was looking at it from the perspective of being called from within `toSvgColor` but totally missed that these can (and I believe are) called independently